ci/conda_recipe/condaversion.patch

```diff
@@ -0,0 +1,14 @@
+diff --git setup.py setup.py
+index 8af8b6d..4e4f9d2 100644
+--- setup.py
++++ setup.py
+@@ -57,6 +57,9 @@
+ import versioneer
+ __version__ = versioneer.get_version()
+ 
++# For conda builds...
++with open("__conda_version__.txt", "w") as f:
```

Comment:
> Why the "needs to be removed"?

I hope it didn't come across as rude - I just wanted to be clear that if we remove this form, we fix the problem you describe in the recipe whereby:
```
# Note also that our setup.py script, which is called by conda-build, writes
+  # a __conda_version__.txt file, so the version number on the binary package
+  # is set dynamically. This unfortunately mean that conda build --output
+  # doesn't really work.
```.